cmd/server/assets/_flash.html

```diff
@@ -1,5 +1,6 @@
 {{define "flash"}}
 <div id="alerts-container" style="position:absolute; top:0; right:1rem; z-index:2000;">
+  {{- if .flash}}
```

Comment:
I'm gonna regret teaching you about dashes, aren't I? 😄

Reply:
mystery whitespace is bs